contracts/GuildBank.sol

```diff
@@ -56,21 +56,21 @@ contract GuildBank is Ownable {
         uint256 lootAmount,
         address[] safeTokenAddresses
     ) public {
-        safeReedemId = safeRedeemId.add(1);
+        safeRedeemId = safeRedeemId.add(1);
 
         uint256 totalLootTokens = lootToken.totalSupply();
 
         require(lootToken.transferFrom(msg.sender, this, lootAmount), "GuildBank::redeemLootTokens - lootToken transfer failed");
 
         // burn lootTokens - will fail if approved lootToken balance is lower than lootAmount
-        lootToken.burn(msg.sender, lootAmount);
+        lootToken.proxyBurn(msg.sender, lootAmount);
```

Comment:
I think this should be a burn, and it should be the GuildBank contract address passed in, not the msg.sender.

Reply:
Updated.